sdk/textanalytics/ai-text-analytics/src/transforms.ts

```diff
@@ -18,75 +24,84 @@ import {
 } from "./models";
 import {
   AnalyzeResponse,
+  AnalyzeTextLROResultUnion,
   AssessmentSentiment,
+  CustomEntityRecognitionLROResult,
+  CustomMultiLabelClassificationLROResult,
+  CustomSingleLabelClassificationLROResult,
   DocumentError,
   EntitiesTaskResult,
+  EntityLinkingLROResult,
   EntityLinkingTaskResult,
+  EntityRecognitionLROResult,
   ErrorModel,
   ErrorResponse,
   EntityLinkingResult as GeneratedEntityLinkingResult,
   EntitiesResult as GeneratedEntityRecognitionResult,
+  HealthcareEntity as GeneratedHealthcareEntity,
+  HealthcareResult as GeneratedHealthcareResult,
   KeyPhraseResult as GeneratedKeyPhraseExtractionResult,
   LanguageDetectionResult as GeneratedLanguageDetectionResult,
   PiiResult as GeneratedPiiEntityRecognitionResult,
   SentenceSentiment as GeneratedSentenceSentiment,
   SentimentResponse as GeneratedSentimentAnalysisResult,
+  HealthcareEntitiesDocumentResult,
+  HealthcareLROResult,
+  HealthcareRelation,
+  HealthcareRelationEntity,
   InnerErrorModel,
+  KeyPhraseExtractionLROResult,
   KeyPhraseTaskResult,
+  KnownAnalyzeTextLROResultsKind,
   LanguageDetectionInput,
   LanguageDetectionTaskResult,
+  PiiEntityRecognitionLROResult,
   PiiTaskResult,
   SentenceTarget,
+  SentimentLROResult,
   SentimentTaskResult,
   TargetRelation,
   TextDocumentInput,
 } from "./generated";
-import { AssessmentIndex, parseAssessmentIndex, sortResponseIdObjects } from "./util";
+import {
+  AssessmentIndex,
+  parseAssessmentIndex,
+  parseHealthcareEntityIndex,
+  sortResponseIdObjects,
+} from "./util";
 import { RestError } from "@azure/core-rest-pipeline";
 
 /**
  * Helper function for converting nested service error into
  * the unified TextAnalyticsError
- * @internal
  */
-export function intoTextAnalyticsError(
-  errorModel: ErrorModel | InnerErrorModel
-): TextAnalysisError {
+function intoTextAnalyticsError(errorModel: ErrorModel | InnerErrorModel): TextAnalysisError {
```

Comment:
nit: Other transform functions in this file use the `to` prefix. Would it make sense to align this function and `makeTextAnalyticsErrorResult` ?